schemas/com.amazon.aws.ec2/instance_identity_document/jsonschema/1-0-0

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema" : "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "self" : {
+    "vendor" : "com.amazon.aws.ec2",
+    "name" : "instance_identity_document",
+    "version" : "1-0-0",
+    "format" : "jsonschema"
+  },
+  "type" : "object",
+  "properties" : {
+    "instanceId" : {
+      "type" : "string",
+      "minLength" : 10,
+      "maxLength" : 10
+    },
+    "devpayProductCodes" : {
+      "type" : [ "array", "null" ],
+      "items" : {
+        "type" : "string"
+      }
+    },
+    "billingProducts" : {
+      "type" : [ "array", "null" ],
+      "items" : {
+        "type" : "string"
+      }
+    },
+    "availabilityZone" : {
+      "type" : "string"
+    },
+    "accountId" : {
+      "type" : "string"
+    },
+    "ramdiskId" : {
+      "type" : [ "string", "null" ],
+      "minLength" : 12,
+      "maxLength" : 12
+    },
+    "architecture" : {
+      "type" : "string"
+    },
+    "instanceType" : {
+      "type" : "string"
+    },
+    "version" : {
+      "type" : "string"
+    },
+    "pendingTime" : {
+      "type" : "string",
+      "format" : "date-time",
+      "minLength" : 20,
```

Comment:
It feels like when we know it's a date-time, the length of the string containing the date-time isn't relevant. Does that make sense?